siriuspy/siriuspy/devices/orbit_interlock.py

```diff
@@ -1,15 +1,142 @@
-"""."""
+"""Orbit Interlock devices."""
 
 import numpy as _np
 
-from .device import Device as _Device
 from ..search import BPMSearch as _BPMSearch
+from ..util import ClassProperty as _classproperty
+from .device import Device as _Device, Devices as _Devices
 
 
-class BPMInterlock(_Device):
-    """BPM Device"""
+class BaseOrbitIntlk:
+    """Base BPM Orbit Interlock Info."""
+
+    CONV_NM2UM = 1e-3  # [nm] --> [um]
+
+    DOWN_2_UP = {
```

Comment:
are these names `UP_2_DOWN` and `DOWN_2_UP` not inverted? are `UP`and `DOWN`prefixes for upstream and downstream?

Reply:
fixed! thanks!